src/util.cpp

```diff
@@ -33,5 +37,13 @@ std::string toHex(const std::vector<uint8_t> &data) {
     return result.str();
 }
 
+std::vector<uint8_t> randomBytes(size_t length) {
+    std::vector<uint8_t> buffer(length);
+    if (1 != RAND_bytes(buffer.data(), buffer.size())) {
+        // TODO: throw?
```

Comment:
I guess that would make sense, yes. Should we have a wrapper around `RAND_bytes` in `openssl_wrap.h`?

Reply:
done